### PR TITLE
Int 1241 test hoi cases and referrals

### PR DESCRIPTION
**Pull request by anuroop15**
Update Acceptance Tests - HoI Card with cases and referrals[#1241](https://osi-cwds.atlassian.net/browse/INT-1241)

### CODE REVIEW

File: spec/support/page/screening_page.rb [hunk 1]
```diff
@@ -144,9 +144,7 @@ def set_decision_attributes(attrs)
   def add_person_from_search(name:, additional_info: nil)
     within '#search-card' do
       autocompleter_fill_in 'screening_participants', "#{name} #{additional_info}"
-      within('ul.react-autosuggest__suggestions-list') do
-        page.first('li', text: name).click
-      end
+      find('strong.highlighted').first('em').click
```
Comment (AmanShurafa) [inline spec/support/page/screening_page.rb]: This is very general and isn't necessary there all the time (some search results have no highlighted keywords). Try to find a better selector that works. For example, all search results has a `profile-picture`. I also see that all search result rows have an id that start with `search-result`.

File: spec/screening/hoi_card_with_cases_and_referrals_spec.rb [hunk 2]
```diff
@@ -38,7 +38,7 @@
   end
 
   it 'does not display duplicate referrals for people who share history' do
-    within '.container' do
+    within all('.container').last do
```
Comment (AmanShurafa) [inline spec/screening/hoi_card_with_cases_and_referrals_spec.rb]: I do not see a necessity for this line altogether. I'm fine with keeping it. But can you at least use `within last('container')` instead.

File: spec/screening/hoi_card_with_cases_and_referrals_spec.rb [hunk 3]
```diff
@@ -72,7 +72,7 @@
       end
     end
 
-    within '.container' do
+    within all('.container').last do
```
Comment (AmanShurafa) [inline spec/screening/hoi_card_with_cases_and_referrals_spec.rb]: Same as above

File: spec/screening/hoi_card_with_cases_and_referrals_spec.rb (hunk 2 above)
Comment (anuroop15) [reply spec/screening/hoi_card_with_cases_and_referrals_spec.rb]: It's not finding it with last('.className'). I also tried fetching element by pseudo element. but it didn't work either. 
Also, this is necessary as it reduces the time complexity of searching three different elements with container class in screening page (page header, cards, main container)

Comment (AmanShurafa) [reply spec/screening/hoi_card_with_cases_and_referrals_spec.rb]: I tried `page.last('.container')` and it's working fine?

**Pull request merged**